README.md

```diff
@@ -384,3 +384,13 @@ mvn deploy:deploy-file -DgroupId=learnositysdk -DartifactId=learnositysdk -Dvers
 ```
 
 5.) Don't forget to commit your file-based repository
+
+## Tracking
+In version v0.16.0 we introduced code to track the following information by adding it to the request being signed:
```

Comment:
Time to add a Changelog.md? https://keepachangelog.com/en/1.0.0/

https://github.com/Learnosity/learnosity-sdk-python/blob/master/ChangeLog.md
https://github.com/Learnosity/learnosity-sdk-php/blob/master/ChangeLog.md

Reply:
Also, I think we should explain why we need this, which is essentially to allow us to roll out updates and changes to SDKs and APIs without risks of disabling features still in use.

Reply:
We have a CHANGELOG.md file in this repo. I've just been adding the changes to a branch that I'll put up when I'm ready to do a release because I hadn't seen the [UNRELEASED] pattern you used in other repos. I can go ahead and add to the CHANGELOG.md changes in this PR if you think it's necessary, otherwise I'll have it in a release PR after this instead.

As for the text of the Tracking details here, we've added this same copy to the Node.js and PHP SDKs already, but we can improve it in the future.

Reply:
The `# UNRELEASED` is a good tag to use, as the release script can just sneak in, replace it with the version number, and create a new entry.

You can add a PR template, too q: https://github.com/Learnosity/learnosity-sdk-python/blob/master/.github/PULL_REQUEST_TEMPLATE.md

(Like literally copying that file and being done with it).

Reply:
I will arrange all of this in the release PR I was planning as I think it would be a bit noisy to continue it on here.